src/main/java/uk/gov/companieshouse/web/accounts/service/transaction/TransactionService.java

```diff
@@ -1,11 +1,10 @@
 package uk.gov.companieshouse.web.accounts.service.transaction;
 
-import uk.gov.companieshouse.api.error.ApiErrorResponseException;
-import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
 
 public interface TransactionService {
 
-    String createTransaction(String companyNumber) throws ApiErrorResponseException;
+    String createTransaction(String companyNumber) throws ServiceException;
```

Comment:
could you have java doc on these methods? and then inherit them in the implementation?

Reply:
Do we need Javadocs here? Only from the perspective that our service methods aren't exposed and the functionality should be easily deduced from the method names.